File: 백준/Bronze/10093. 숫자/숫자.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String s = br.readLine();
        StringTokenizer st = new StringTokenizer(s);
        
        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());

        if (a == b) {
            System.out.println(0);
        } else {
            if (a > b) {
                int temp = a;
                a = b;
                b = temp;
            }
            int cnt = 0;
            String answer = "";
            for (int i=a+1; i<b; i++) {
                cnt++;
                answer += i + " ";
            }
            System.out.println(cnt);
            System.out.println(answer);
        }
    }
}

File: 백준/Bronze/10093. 숫자/숫자.swift
import Foundation

var ab = readLine()!.split(separator: " ").map{Int(String($0))!}
ab.sort()

if (ab[0] == ab[1]) {
    print(0)
} else {
    var cnt = 0
    var answer = ""
    for n in (ab[0]+1)..<ab[1] {
        cnt += 1
        answer += "\(n) "
    }
    print(cnt)
    print(answer)
}